packages/react-app-revamp/config/supabase/index.ts

```diff
@@ -1,6 +1,12 @@
+import { isSupabaseConfigured } from "@helpers/database";
 import { createClient } from "@supabase/supabase-js";
 
-const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL as string;
-const supabaseAnonKey = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY as string;
+var supabaseUrl: string = "000000000000000000000000000000000000.0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.0000000000000000000000000000000000000000000";
+var supabaseAnonKey: string = "000000000000000000000000000000000000.0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.0000000000000000000000000000000000000000000";
```

Comment:
Any reason why we set `var` to be the type of string? I'm thinking if we should also declare it as `let` for the sake of consistency.

I know that TS is capable of inferring the type automatically but not sure if you are setting it for explicit reasons?

Reply:
`let` and removing the explicit typing sounds good to me if that's best practice! I'll go ahead and make those changes!